src/Toaster/Toast.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { ToastType } from 'react-toastify';
+import { IToast } from '.';
+import { Box, IBoxCSS } from '../Box';
+import { Icon } from '../Icon';
+import { Text } from '../Text';
+
+export const iconMap = {
+  [ToastType.INFO]: 'info-circle',
+  [ToastType.WARNING]: 'exclamation-triangle',
+  [ToastType.ERROR]: 'times-circle',
+  [ToastType.SUCCESS]: 'check-circle',
+  [ToastType.DEFAULT]: 'lightbulb',
+};
+
+export const Toast = React.forwardRef<HTMLElement, IToast>((props, ref) => {
+  const { content, type, css, closeToast, ...rest } = props;
```

Comment:
`closeToast` seems to be unused. Is it intentionally removed from props?

Reply:
This is intentionally left unused. See {...rest} in the <Box>? If I remove 'closeToast' then it will become part of 'rest' and will result in nasty console errors (Box is not happy about closeToast prop)

Reply:
👍